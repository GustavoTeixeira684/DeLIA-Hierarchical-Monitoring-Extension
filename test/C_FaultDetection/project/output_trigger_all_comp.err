EXEC_TEST: no process found
