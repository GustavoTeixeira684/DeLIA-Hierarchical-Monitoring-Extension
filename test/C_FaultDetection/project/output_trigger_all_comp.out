removed './data/DeLIA_Iteration_Localnode_0.txt'
removed './data/DeLIA_Iteration_Localnode_1.txt'
removed './data/DeLIA_Iteration_Localnode_2.txt'
removed './data/DeLIA_Iteration_Localnode_3.txt'
removed './data/DeLIA_Iteration_Localnode_4.txt'
removed './data/DeLIA_Iteration_Localnode_5.txt'
removed './data/DeLIA_Iteration_Localnode_6.txt'
removed './data/DeLIA_Iteration_Localnode_7.txt'
removed './data/DeLIA_Iteration_Localnode_8.txt'
removed './data/DeLIA_Iteration_Localnode_9.txt'
removed './data/DeLIA_Local_node_0_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_1_gIteration_4_local_data_example.bin'
removed './data/DeLIA_Local_node_2_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_3_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_4_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_5_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_6_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_7_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_8_gIteration_5_local_data_example.bin'
removed './data/DeLIA_Local_node_9_gIteration_5_local_data_example.bin'
removed './data/DeLIA_gIteration.txt'
removed './data/global_data_example.bin'
removed './data/settings.bin'
DELETING Data
CI TEST
RANK_0_RECEIVED_SIGNAL
RANK_1_RECEIVED_SIGNAL
RANK_2_RECEIVED_SIGNAL
RANK_3_RECEIVED_SIGNAL
RANK_4_RECEIVED_SIGNAL
RANK_5_RECEIVED_SIGNAL
RANK_6_RECEIVED_SIGNAL
RANK_7_RECEIVED_SIGNAL
RANK_8_RECEIVED_SIGNAL
RANK_9_RECEIVED_SIGNAL
